### PR TITLE
Review.java

### PR DESCRIPTION
**Pull request by JordanTheDeveloper**
The totalSenitment method finds the sentiment value of each word. Letters are then added together and then stops when the method reaches a space. A method is then called that finds the sentiment value of the entire word. Next the value is stored and added into sentiment, finally the value is returned by totalSentiment.

@kelseajacobs is this correct?
@Briyannis is this correct?
@xaamaro is this correct?

### CODE REVIEW

File: Review.java [hunk 1]
```diff
@@ -165,4 +165,52 @@ public static String randomAdjective()
...
+		}
+		else
+		{
+			removePunctuation(word);
+			sentiment = sentiment + sentiment(word);
+			word = “ “;
+		}
+	}
+	return sentiment;
+}
```
Comment (kelseajacobs) [inline Review.java:188]: @JordanTheDeveloper looks good!

**Review by kelseajacobs — COMMENTED** [review]
@JordanTheDeveloper this looks great. you should go ahead and merge it to finish the assignment.

File: Review.java [hunk 2]
```diff
@@ -165,4 +165,52 @@ public static String randomAdjective()
...
+			word = word + letter;
+		}
+		else
+		{
+			removePunctuation(word);
+			sentiment = sentiment + sentiment(word);
+			word = “ “;
+		}
+	}
+	return sentiment;
```
Comment (Briyannis) [inline Review.java:187]: This looks good!

**Review by Briyannis — COMMENTED** [review]
This looks good!

**Comment by xaamaro** [comment]
it looks nice. Just make sure you add comments as you go along so that other coders can know what you're talking about

**Pull request merged**